components/formComponents/DateTimePicker.tsx

```diff
@@ -73,16 +61,6 @@ export type DateTimePickerProps = {
    * hour and minute information.
    */
   formatTime?: (date: Date) => string
-
-  /**
-   * The minimum date for this picker.
-   */
-  minimumDate?: Date
```

Comment:
I've temporarily removed it for now bc it was acting weird on Android and we didn't really need the functionality ASAP. Also I kinda want to redo the UI for the form date time picker, but I didn't implement it fully here since it would take a non-trivial amount of time.